scalding-core/src/main/scala/com/twitter/scalding/typed/Sketched.scala

```diff
@@ -91,9 +91,9 @@ case class Sketched[K, V](pipe: TypedPipe[(K, V)],
 case class SketchJoined[K: Ordering, V, V2, R](left: Sketched[K, V],
   right: TypedPipe[(K, V2)],
   numReducers: Int)(joiner: (K, V, Iterable[V2]) => Iterator[R])
-  extends HasReducers {
+  extends MustHasReducers {
```

Comment:
this is both, right? HasReducers and MustHasReducers? Or MustHasReducers should extend HasReducers.